src/github/deployment.ts

```diff
@@ -1,30 +1,97 @@
 import transformDeployment from "../transforms/deployment";
 import { emitWebhookProcessedMetrics } from "../util/webhooks";
 import { CustomContext } from "./middleware";
-import { DeploymentsResult } from "../jira/client";
+import getJiraClient, { DeploymentsResult } from "../jira/client";
+import { booleanFlag, BooleanFlags } from "../config/feature-flags";
+import sqsQueues from "../sqs/queues";
+import { GitHubAPI } from "probot";
+import { WebhookPayloadDeploymentStatus } from "@octokit/webhooks";
+import { LoggerWithTarget } from "probot/lib/wrap-logger";
 
-export default async (context: CustomContext, jiraClient): Promise<void> => {
-	const jiraPayload = await transformDeployment(context.github, context.payload, jiraClient.baseURL, context.log);
+export default async (context: CustomContext, jiraClient, _util, githubInstallationId: number): Promise<void> => {
+
+	if (await booleanFlag(BooleanFlags.USE_SQS_FOR_DEPLOYMENT, false, jiraClient.baseURL)) {
+		await sqsQueues.deployment.sendMessage({
+			jiraHost: jiraClient.baseURL,
+			installationId: githubInstallationId,
+			webhookPayload: context.payload,
```

Comment:
So, we are confident that these payloads are < 256kb?

Reply:
yes. The metrics show that a deployment_status events is max 13 KB (there is no unbounded content in it like in PR descriptions or issue descriptions)